changelogs/unreleased/gh-10304-allow-specific-function-access.md

```diff
@@ -0,0 +1,4 @@
+## feature/box
```

Comment:
Let's use `config` component here and in commit message/pull request.

Reply:
corrected